robocop/checkers/spacing.py

```diff
@@ -42,7 +42,7 @@ def parse_file(self):
         super().parse_file()
         if self.raw_lines:
             last_line = self.raw_lines[-1]
-            if last_line == '\n':
+            if last_line in ['\n', '\r']:
```

Comment:
What about common line ending for Windows - '\r\n'?

Reply:
I add a new commit to fix this